gems/quilt_rails/lib/quilt_rails/engine.rb

```diff
@@ -0,0 +1,5 @@
+module Quilt
+  class Engine < ::Rails::Engine
+    isolate_namespace QuiltRails
```

Comment:
does this prevent consumers from using the mixin directly?

Reply:
In my manual tests, no. You can still do everything the same as whats in the current readme.md.